### DISCUSSION TITLE
Headphone option for Radio - Audio

### DISCUSSION THREAD

**Discussion by davidsteelerose** [AlexandreRouma/SDRPlusPlus#1530] [General]
<img width="291" alt="image" src="https://github.com/user-attachments/assets/279ac19a-72a5-4ccb-9407-6c08e88b46d4">

How do I add additional Radio - Audio options. I would like to listen using Headphones. I'm on a Mac and I can select headphones in the settings, but it still plays through the speakers.

**Comment by joekuzak** [comment]
<img width="297" alt="Screenshot 2024-11-20 at 11 28 24" src="https://github.com/user-attachments/assets/4501758c-fdf8-455f-8178-f33e2776e464">
Whatever you select in the audio drop down (Top Right of screen) will appear in your SDR++ Radio.

<img width="351" alt="Screenshot 2024-11-20 at 11 31 55" src="https://github.com/user-attachments/assets/cbcc9a33-07ed-409b-b9b1-e7294397b71c">

You need to restart SDR++ to see it.

**Comment by davidsteelerose** [comment]
Thank you. That works!On Nov 20, 2024, at 6:33 AM, joekuzak ***@***.***> wrote:﻿
Screenshot.2024-11-20.at.11.28.24.png (view on web)
Whatever you select in the audio drop down (Top Right of screen) will appear in your SDR++ Radio.
Screenshot.2024-11-20.at.11.31.55.png (view on web)
